src/sets.jl

```diff
@@ -435,6 +435,53 @@ Base.isapprox(a::T, b::T; kwargs...) where {T <: Union{SOS1, SOS2}} = isapprox(a
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+	ActivationCond
+
+Activation condition for an indicator constraint.
+Used as first type parameter of `IndicatorSet{ActiveCond,S}`.
+"""
+@enum ActivationCond begin
```

Comment:
Why not call it `ActivationCondition` in full ?

Reply:
changed